kubernetes_state/metadata.csv

```diff
@@ -54,6 +54,10 @@ kubernetes_state.nodes.by_condition,gauge,,,,To sum by `condition` and `status`
 kubernetes_state.hpa.min_replicas,gauge,,,,Lower limit for the number of pods that can be set by the autoscaler,0,kubernetes,k8s_state.hpa.min_replicas
 kubernetes_state.hpa.max_replicas,gauge,,,,Upper limit for the number of pods that can be set by the autoscaler,0,kubernetes,k8s_state.hpa.max_replicas
 kubernetes_state.hpa.desired_replicas,gauge,,,,Desired number of replicas of pods managed by this autoscaler,0,kubernetes,k8s_state.hpa.desired_replicas
+kubernetes_state.pdb.pods_desired,gauge,,,,Minimum desired number of healthy pods,0,kubernetes,k8s_state.pdb.pods_desired
+kubernetes_state.pdb.disruptions_allowed,gauge,,,,Number of pod disruptions that are currently allowed,0,kubernetes,k8s_state.pdb.disruptions_allowed
+kubernetes_state.pdb.pods_healthy,gauge,,,,Current number of healthy pods,0,kubernetes,k8s_state.pdb.pods_healthy
```

Comment:
While other gauges' `0` orientation makes sense, this one might deserves a `1`, wdyt?

Reply:
Yep, good catch 👍